src/CommunityToolkit.Aspire.Hosting.SqlDatabaseProjects/README.md

```diff
@@ -48,6 +48,21 @@ builder.AddSqlProject("mysqlproj")
 builder.Build().Run();
 ```
 
+## Support for existing databases
+Instead of using the `AddDatabase` method to create a new database, you can specify an existing database:
```

Comment:
I'm wondering if `database` is a bit of misnomer here. We're specifically talking about the server here I believe.

Reply:
I think a database, as the database in the connection string will be used (or created if needed)

Reply:
Actually, I will change it!